js/src/views/Dapps/dapps.css

```diff
@@ -29,3 +30,25 @@
   flex: 0 1 50%;
   box-sizing: border-box;
 }
+
+.overlay {
+  background: rgba(0, 0, 0, 0.85);
+  bottom: 0.5em;
+  left: -0.25em;
```

Comment:
This looks off on my screen (h-scroll bars appears). I think it should be `0.125em` instead